client/blocks/comment-detail/comment-detail-reply.jsx

```diff
@@ -2,18 +2,100 @@
  * External dependencies
  */
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import AutoDirection from 'components/auto-direction';
+import { getCurrentUser } from 'state/current-user/selectors';
 
 export class CommentDetailReply extends Component {
+	state = {
+		commentText: '',
+		hasFocus: false,
+	};
+
+	handleTextChange = event => this.setState( { commentText: event.target.value } );
+
+	setFocus = () => this.setState( { hasFocus: true } );
+
+	submit = () => {
+		const comment = {
+			authorAvatarUrl: this.props.currentUser.avatar_URL,
+			authorName: this.props.currentUser.display_name,
+			authorUrl: this.props.currentUser.primary_blog_url,
```

Comment:
We're very unlikely to hit this in a handler but `getCurrentUser` can return null. Whenever I see more than one property access, I like to bulletproof.